src/Audio/Listener.cs

```diff
@@ -69,28 +69,28 @@ public static Vector3f UpVector
         }
         
         #region Imports
-        [DllImport("csfml-audio-2", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
Do you have any measurements on the performance difference of removing `SuppressUnmanagedCodeSecurity`?

SFML has always been for "no unnecessary optimizations", so since this was added in the first place, it's likely here for a reason.

If the performance difference is minimal though, I don't see why removing it is a problem!